boards/mega-xplained/doc.txt

```diff
@@ -10,27 +10,55 @@ is an evaluation kit by Atmel (now Microchip) for their ATmega1284P microcontrol
 
 ### Flash the board
 
-1. The board may be flashed through JTAG or using a SPI ISP programmer. If
-   the Buspirate is being used, then `make flash` can be used to flash the
-   board:
-```
-     make BOARD=mega-xplained -C examples/hello-world flash
-```
+You can change the fuses and flash the board using an SPI ISP programmer such
+as the Bus Pirate, or using a JTAG programmer such as the Atmel-ICE with a
+JTAG adapter.
+
+If using the Atmel-ICE as a JTAG programmer, connect the adapter as shown in
+the image below.
+
+   [photo](https://photo.jpg)
```

Comment:
I'd say here you might reference something existent, git-lfs will take some time to be merged and this already can help to test out this board.

Reply:
No need for a picture. The board and the JTAG have one square pin, just mention to match the squared one.